styles.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+.body {
+  position: relative;
+}
+
+.header {
+  background: #3c3a39;
+  padding: 0 30px;
+  height: 10vh;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  align-items: center;
+  position: fixed;
+  z-index: 1;
+  font-family: "Roboto", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+}
+
+.logo {
+  color: #fff;
+}
+
+.main-section {
+  background-color: #1c1a19;
+  position: relative;
+  height: 100vh;
+  padding: 120px 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.image {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.img {
+  background-image: url("./images/header-image.png");
+  background-position: right bottom -20%;
+  background-repeat: no-repeat;
+}
+
+li {
+  display: inline;
+}
+
+h1 {
```

Comment:
**Optional**
Kindly keep in mind that following the given design is so important for you in best practices and also for making the website professional, consistent, and good looking.
You should follow the design template in typography, font size, font weight, colors, images, and spacings (margin and padding) 
_Take a look at the images down below._
> I'm sorry this didn't show up for you in the earlier review.
### **Your design**
![Screenshot_2021-06-10-02-07-13-26](https://user-images.githubusercontent.com/47625407/121445043-f6eaf900-c990-11eb-9ed5-23446d6f9dab.jpg)
### **[Original design](https://www.figma.com/file/NWVDbyHlxMtZO5VX92hpM1/Microverse-Student-Project-1-Copy?fuid=892210429043439399)**
![Screenshot_2021-06-10-02-06-57-10](https://user-images.githubusercontent.com/47625407/121445066-02d6bb00-c991-11eb-96e8-e3bc63eab0c7.jpg)